lib/ipa_test_kit/custom_groups/v1.0.0/ipa_smart_launch_group.rb

```diff
@@ -81,10 +92,17 @@ class IPASMARTLaunchGroup < Inferno::TestGroup
             title: 'SMART Public Standalone Launch',
             config: {
               inputs: {
-                client_secret: {
-                  default: nil,
-                  locked: true,
-                  optional: true
+                standalone_smart_auth_info: {
```

Comment:
This should be `smart_auth_info`, none of this configuration is actually being applied.